.github/workflows/build_and_test.yml

```diff
@@ -15,8 +15,14 @@ jobs:
     - name: Check out repo
       uses: actions/checkout@v2
 
+    - name: Clone dependency packages
+      run: mkdir dependency_ws && vcs import dependency_ws < build_depends.repos
+
     - name: Install missing dependencies
-      run: rosdep update && DEBIAN_FRONTEND=noninteractive sudo rosdep install --from-paths . --ignore-src --rosdistro foxy -y
+      run: |
```

Comment:
I'm not sure what the `|` does in this situation? In the workflow for Pilot.Auto this appears in all the run statements so should the run statements below also have this?

Reply:
`|` can be used to specify multiple commands.
I thought it would improve readability.